Project report/full-report.tex

```diff
@@ -70,15 +73,15 @@
     \makeack
 
     \newpage
-    \addcontentsline{toc}{chapter}{สารบัญ}
+    \addcontentsline{toc}{chapter}{\contentsname{}}
```

Comment:
I think we can remove the ending brackets of `\contentsname{}`

Reply:
Fixed @DobaKung